.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
We are using 'stadndard' as a linter. 
We better add exception to the package.json instead of adding another linter (see: https://standardjs.com/)

Reply:
Thanks for reacting so fast. I have made the fixes, and created a Merge Request on the Gitlab repository (I mistakenly thought this repo was the active one at first)

see: https://gitlab.com/xsellier/joi4express/merge_requests/2/diffs

Thanks